next.config.js

```diff
@@ -1,4 +1,12 @@
 module.exports = {
   // assetPrefix can differ based on NODE_ENV in case its not served from domain root
   assetPrefix: '',
+  webpack: (config) => {
+    config.module.rules.push({
+      test: /\.svg$/,
+      use: ['@svgr/webpack'],
+    })
+
+    return config
+  },
```

Comment:
Enables importing and using svgs like components (I find this cleaner than including them in <img/> tags). Let me know if this goes against some practices that are already in place

Example
```jsx
import SampleSvg from 'public/sample.svg'

// then to use...
<SampleSvg />
```

Reply:
All good